Projects/My_website/myWebsite/account/utils.py

```diff
@@ -0,0 +1,19 @@
+import re
+
+
+# function for validation of strong password
+def StrongPasswordValidator(password):
```

Comment:
@codal-uchoudhary please use snack cases. this func name should be `strong_password_validator`